src/contracts/interfaces/IDelegationManager.sol

```diff
@@ -147,6 +153,15 @@ interface IDelegationManager is ISignatureUtils {
     /// @notice Emitted when the `withdrawalDelayBlocks` variable is modified from `previousValue` to `newValue`.
     event WithdrawalDelayBlocksSet(uint256 previousValue, uint256 newValue);
 
+    /// @notice Emitted when an operator's registration status for an AVS is updated
+    event OperatorRegistrationStatusUpdatedForAVS(address indexed operator, address indexed avs, OperatorRegistrationStatus status);
```

Comment:
maybe just name this something like `AVSRegistrationStatusUpdated`? idk if that's actually that much better tho